test/parallel/test-repl-throw-null-or-undefined.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+require('../common');
+
+// This test ensures that the repl does not 
+// crash or emit error when throwing `null|undefined`
+// ie `throw null` or `throw undefined`
+
+const assert = require('assert');
+const repl = require('repl');
+
+const r = repl.start();
+
+r.write('throw null\n');
+r.write('throw undefined\n');
+
+let i = 0;
+r.on('line', (statement) => {
+  const isThrowNull = statement.includes('null');
+  const assertFail = 'repl did not throw ';
```

Comment:
Nit: can you please remove the trailing space? There is already one in the template string below.